cli/provider/cmd.go

```diff
@@ -203,6 +224,21 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command) error {
 		}
 
 	case "record", "test", "rerecord":
+		if cmd.HasAlias("validate") {
```

Comment:
why is this code here, it is not related to record, test or rerecord commands?

Reply:
as when we pass the cmd to **AddFlags** function , keploy contract test command's name will be "test" so it will enter this case so I made an Alias to differentiate between the keploy test command and keploy contract test command when entering this case.